### ISSUE TITLE
jogwheel edge/tempo nudge

### ISSUE THREAD

**Issue by errolkokbas** [rasteri/SC1000#24]
Would it be possible to implement some sort of nudging logic to the jogwheel if no touch is detected? This way you could make the outside edge of the platter non-conductive somehow, like perhaps removing some of the gold platting on the pcb around the outer edge, to facilitate using the nudge logic without the need for any other button presses. 

To be clear, what I mean by nudge is that when you move the jog wheel, the tempo changes briefly, with a rapid but gradual change, and then goes back to what it was again. This is super useful for beat matching tracks, and would compliment the pull request for LED ring indicators nicely, more and more like a real turntable. (I really want to use two sc500's with a serato timecode signal audio file for dvs)

**Comment by rasteri** [comment]
Yeah like how a CDJ works, I getcha. I'll possibly implement it in a future release.

**Comment by errolkokbas** [comment]
Just curious, would this be strictly a firmware update, besides making a non-conductive outside edge on the platter? I am thinking it could even be done by applying a layer of paint or something similar. I am wondering because I want to know if I should wait for a new board revision or other changes to the hardware, before I buy a set of sc500's. Or, could I go ahead and get them now and just update them in the future?

**Comment by errolkokbas** [comment]
Just curious, would this be strictly a firmware update, besides making a non-conductive outside edge on the platter? I am thinking it could even be done by applying a layer of paint or something similar. I am wondering because I want to know if I should wait for a new board revision or other changes to the hardware, before I buy a set of sc500's. Or, could I go ahead and get them now and just update them in the future?

**Comment by errolkokbas** [comment]
Whoops, didn't mean to close this issue, still learning github over here....my browser loaded the page with the text of my previous post duplicated in the comment box, I thought I was clearing that.
